syntax = "proto3";
package pkg;





/*
BatchUpdatePackagePermission请求
*/
message BatchUpdatePackagePermissionRequest {
    /*
    需要变动的权限
    */
    repeated string permissions = 1 ;
    /*
    需要权限变动的用户
    */
    repeated string authorizers = 2 ;
    /*
    变动的方法
    */
    string method = 3 ;
    /*
    变动的包id
    */
    repeated string packageIds = 4 ;
}

/*
BatchUpdatePackagePermission返回
*/
message BatchUpdatePackagePermissionResponse {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回信息
    */
    string message = 2 ;
    /*
    包Id
    */
    string packageId = 3 ;
}

/*
BatchUpdatePackagePermissionApi返回
*/
message BatchUpdatePackagePermissionResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    repeated pkg.BatchUpdatePackagePermissionResponse data = 4 ;
}

